frontend/src/app/components/containers/complaints/webeoc-complaint-updates/webeoc-complaint-update-list.tsx

```diff
@@ -41,7 +41,7 @@ export const WebEOCComplaintUpdateList: FC<Props> = ({ complaintIdentifier }) =>
   return (
     <>
       {complaintUpdates && complaintUpdates.length > 0 && (
-        <div className="comp-complaint-details-block">
+        <div className="comp-container comp-complaint-details-block">
           <div>
             <h6>Complaint Updates ({complaintUpdates.length})</h6>
```

Comment:
You'll probably want to vet this with the design team, however, based on the current page hierarchy this should probably be an H2 element, and ensure the spacing/visual appears similar to both the 'Complaint Details' section and the 'Outcome Report' section.

Second, the structure (from a semantics perspective) should match (see complaint-details-edit.tsx - line 675 for structure).

Reply:
Semantics can be resolved in a subsequent ticket as this ticket is only to accommodate the container width.